pkgs/development/node-packages/node-packages.nix

```diff
@@ -328,15 +328,6 @@ let
         sha512 = "qrXI4+S8W7IF6e1nlDYX2KfdzxGHyAOj5kGvWk+TqBuAnA0rWQ513hJzdviiGpbB5VPnJkEhOVsDets8acKd6w==";
       };
     };
-    "@astrojs/compiler-1.8.2" = {
```

Comment:
I removed all of these packages manually, and it changes nothing else in this file, so it shouldn't have any merge conflicts. I'll rebase to current master commit so that there's less chance of such.